_data/docs.yml

```diff
@@ -172,6 +172,17 @@ libs:
       legacy: 1
       stable: 1
       nightly: 1
+  libcuspatial:
+    name: libcuspatial
+    path: libcuspatial
+    desc: 'libcuspatial is a GPU accelerated C/C++ library for accelerating GIS workflows including point-in-polygon, spatial join, coordinate systems, shape primitives, distances, and trajectory analysis.'
```

Comment:
Should we keep the description the same as: https://github.com/rapidsai/cuspatial/blob/branch-22.08/cpp/doxygen/main_page.md ?

Reply:
sure! Didn't see that. updated in dc61a5a